cmd/mms/commands.go

```diff
@@ -53,6 +53,7 @@ func subscribeEvents(hubs []productionHub) func(*cli.Context) error {
 	}
 }
 
+//TODO: Split in two functions so that it is callable both from cli and the Python binding
```

Comment:
Don't think this is necessary anymore. I think I will hook directly into `pkg/mms`.